sdk/data/azcosmos/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+* Fixed sending `Prefer` header on metadata operations.
```

Comment:
Shall we call out the full header name to make it clear? `PreferReturnMinimal`?
Or may be we can call out the API name so that we can track this change later? like `contentResponseOnWrite`?

Reply:
Its for all APIs. The header name is `Prefer`, but I added the value for clarity.